apps/admin/src/modules/configurations/compositions/metadata.ts

```diff
@@ -2,7 +2,8 @@ import { reactive } from 'vue'
 import { useEditedInputModels } from '../../core/compositions/edited-inputs'
 
 const configurationKeysTypes = {
-	logoUrl: ['text', 'upload']
+  logoUrl: ['text', 'upload'],
```

Comment:
can dry a bit if you want 

```
const commonConfig = ['text', 'upload'] 
const configurationKeysTypes = {
  logoUrl: commonConfig,
  themeStylesUrl: commonConfig,
  }
```

Reply:
The common is leaving it empty, so it will use basic text input. :)

Reply:
done.